tests/integration/udf/test_count_dict_num_unique.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 
 
-@pytest.mark.parametrize("source_type", ["snowflake, spark"], indirect=True)
+@pytest.mark.parametrize("source_type", ["snowflake"], indirect=True)
```

Comment:
Is there a reason this isn't using `SNOWFLAKE_SPARK_DATABRICKS`?

Reply:
there was a test failure when i enabled that. i'll fix in a follow-up!